src/librustc_mir/interpret/terminator.rs

```diff
@@ -67,6 +68,7 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> InterpretCx<'a, 'mir, 'tcx, M>
                 ref func,
                 ref args,
                 ref destination,
+                cleanup,
```

Comment:
Everything else uses `ref` here... I don't really know why but it seems reasonable to be consistent?

Reply:
Ok